firmware/application/apps/ui_recon.cpp

```diff
@@ -1350,14 +1350,12 @@ size_t ReconView::change_mode(freqman_index_t new_mod) {
                                                    u"AUTO_AUDIO_", u"AUDIO",
                                                    RecordView::FileType::WAV, 4096, 4);
     }
-
+    add_child(record_view.get());
```

Comment:
This was one I was worried about. A lot of those method expect the widget has a non-null parent. Asserts in widget.cpp would be really helpful.